tests/testproject/TestProject.IntegrationServiceA/Program.cs

```diff
@@ -2,7 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 var builder = WebApplication.CreateBuilder(args);
-builder.AddSqlServerClient("sql");
+builder.AddSqlServerClient("sqlserver");
```

Comment:
Consider adding another integration project that has multiple resources of each type

Reply:
This is more of a smoke test than an exhaustive test. We also need to consider resources on the CI machines once we eventually get these tests running in that environment.